README.md

```diff
@@ -4,3 +4,33 @@ This repository holds the Go code used by the `hashicorp/consul-ecs` Docker imag
 This image is used to help with the installation and operation of Consul on ECS.
 
 See [https://consul.io/docs/ecs](https://consul.io/docs/ecs) for more information.
+
+## Consul on Roadmap
+
+Knowing about our upcoming features and priorities helps our users plan. This repository contains information about what we are working on and allows all our user to give direct feedback.
+
+
+[See the roadmap »](https://github.com/hashicorp/consul-ecs/projects/1)
+
+## Roadmap FAQs
+**Why did you build this public facing roadmap?**
+
+We know that our customers are making decisions and plans based on what we are developing, and we want to provide our customers the insights they need to plan.
+
+**Why does your roadmap not have specific dates?**
+
+Because the highest priority for us to put out a secure and operationally stable product, we can't provide specific target dates for features. The roadmap is subject to change at any time, and roadmap issues in this repository do not guarantee a feature will be launched as proposed.
```

Comment:
```suggestion
Because the highest priority for us is to put out a secure and operationally stable product, we can't provide specific target dates for features. The roadmap is subject to change at any time, and roadmap issues in this repository do not guarantee a feature will be launched as proposed.
```